pgxn/neon/neon.c

```diff
@@ -366,23 +366,38 @@ RestoreRunningXactsFromClog(CheckPoint *checkpoint, TransactionId **xids, int *n
 	if (!TransactionIdIsValid(checkpoint->oldestActiveXid))
 	{
 		elog(LOG, "cannot restore running-xacts from CLOG because oldestActiveXid is not set");
-		goto fail;
-	}
+		if (running_xacts_overflow_policy == OP_WAIT)
```

Comment:
Hmm, on second thought, I don't understand why we check the `running_xacts_overflow_policy` here at all. At this moment, we do not know yet whether there is an overflow or not.

Maybe that's valid, but then it's worth of a comment

Reply:
Sorry, I didn't want to rename GUC name and variable. And didn't want to introduce yet another GUC.
So I used the same GUC `running_xacts_overflow_policy` to handle situation when oldestActiveXid is invalid.